### PR TITLE
added button Reset Overrides for filters

### PR DESCRIPTION
**Pull request by dariaivanciucova**
## Description
Added a simple tool that reads the filters in the active view and, for selected filters, resets their overrides.

## Changes
Added 1 pushbutton to the Project > Reset Filters dropdown.

## Related Issues
None

## Checklist
- [x] Code follows the project's coding style
- [ ] Tests have been added or updated to cover the changes
- [x] Documentation has been updated to reflect the changes
- [ ] All automated tests are passing
- [ ] Reviewed by at least one team member

## Screenshots (if applicable)

image
Remove overrides

### CODE REVIEW

File: pyChilizer.tab/Project.panel/colorizers2.stack/Reset Filters.pulldown/Reset Overrides.pushbutton/script.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+def overrides_reset():
+
+    # returns Override Graphic Settings object with all overrides set to initial values
+    no_color = DB.Color.InvalidColorValue
+    invalid_id = DB.ElementId.InvalidElementId
+
+    overrides = DB.OverrideGraphicSettings()
```
Comment (jmcouffin) [inline pyChilizer.tab/Project.panel/colorizers2.stack/Reset Filters.pulldown/Reset Overrides.pushbutton/script.py:20]: you can simplify: 


```python
# erase overrides
override = DB.OverrideGraphicSettings().Dispose()
# get clear graphics without overrides
override= DB.OverrideGraphicSettings()
```

and maybe this only is enough

`override= DB.OverrideGraphicSettings()`

Comment (dariaivanciucova) [reply pyChilizer.tab/Project.panel/colorizers2.stack/Reset Filters.pulldown/Reset Overrides.pushbutton/script.py:16]: fab, thanks for the tip JM! 🙌

Comment (dnenov) [reply pyChilizer.tab/Project.panel/colorizers2.stack/Reset Filters.pulldown/Reset Overrides.pushbutton/script.py:16]: Nice one! Thanks @jmcouffin  Closing now

**Review by dnenov — APPROVED** [review]
LGTM

**Pull request merged**